samples/extensions/dynamic_rendering/dynamic_rendering.cpp

```diff
@@ -380,6 +426,16 @@ void DynamicRendering::build_command_buffers()
 			vkCmdBeginRenderingKHR(draw_cmd_buffer, &render_info);
 			draw_scene();
 			vkCmdEndRenderingKHR(draw_cmd_buffer);
+
+			vkb::insert_image_memory_barrier(draw_cmd_buffer,
+			                                 swapchain_buffers[i].image,
+			                                 VK_ACCESS_SHADER_WRITE_BIT,
+			                                 VK_ACCESS_COLOR_ATTACHMENT_READ_BIT,
+			                                 VK_IMAGE_LAYOUT_COLOR_ATTACHMENT_OPTIMAL,
+			                                 VK_IMAGE_LAYOUT_PRESENT_SRC_KHR,
+			                                 VK_PIPELINE_STAGE_FRAGMENT_SHADER_BIT,
```

Comment:
`VK_PIPELINE_STAGE_COLOR_ATTACHMENT_OUTPUT_BIT` (and accessMask `VK_ACCESS_COLOR_ATTACHMENT_WRITE_BIT`). And the dstAccessMask should be 0 (present semaphore takes care of everything here)